### DISCUSSION TITLE
Sync between start en end time

### DISCUSSION THREAD

**Discussion by TTB82** [phw198/OutlookGoogleCalendarSync#1900] [Ideas]
Add option to sync only between starttime A and endtime B. For instance, sync only between 8:30 (AM) and 17:00.
Reason: At night sometimes the sync generates errors and stops syncing (manuel interaction is needed making a choice to sync or not). This could prevent this from happening. Furthermore in my case syncing between 17:00 and 8:30 isn't necessary (no new/changed items).

**Comment by phw198** [comment]
Thanks for the idea, but it's a duplicate of #1615 so I'll close this one.
